spec/features/course/achievement_management_spec.rb

```diff
@@ -164,15 +189,26 @@
         expect(page).not_to have_selector('tr.condition > td.level-condition-content', text: '1')
       end
 
-      def create_achievement_condition
-        other_achievement = create(:course_achievement, course: course)
-        achievement_condition =
-          create(:course_condition_achievement,
-                 course: course, achievement: other_achievement)
-        achievement = create(:course_achievement,
-                             course: course,
-                             conditions: [achievement_condition])
-        [achievement_condition, achievement, other_achievement]
+      def add_achievement_to_course(achievement, course)
```

Comment:
Unused method argument - `achievement`. If it's necessary, use `_` or `_achievement` as an argument name to indicate that it won't be used.